parser.js

```diff
@@ -43,6 +43,8 @@ define(function(require) {
             error = {};
             error.message = render(errorMessages[result.error.type], result.error);
             error.cursor = result.error.cursor;
+            //error.start = result.error.openTag.start;
```

Comment:
Don't leave commented code (I've written on this here http://blog.humphd.org/vocamus-894/)